tests/client/disco/components/TestAddon.js

```diff
@@ -289,18 +293,29 @@ describe('<Addon />', () => {
     beforeEach(() => {
       previewTheme = sinon.spy();
       resetPreviewTheme = sinon.spy();
-      const data = { ...result, type: ADDON_TYPE_THEME, previewTheme, resetPreviewTheme };
+      const data = {
+        ...result,
+        type: ADDON_TYPE_THEME,
+        previewTheme,
+        resetPreviewTheme,
+      };
       root = renderAddon({ addon: data, ...data });
       themeImage = findDOMNode(root).querySelector('.theme-image');
     });
 
-    it('runs theme preview onMouseOver on theme image', () => {
-      Simulate.mouseOver(themeImage);
+    afterEach(() => {
+      unmountComponentAtNode(findDOMNode(root));
```

Comment:
AFAIK usually components don't need to be unmounted in tests. renderIntoDocument doesn't _actually_ render them into a document, it just creates a div and renders into that (https://github.com/facebook/react/blob/master/src/test/ReactTestUtils.js#L128). This means that normally, it all just gets garbage collected and we don't have to worry about it. However the HoverIntent component creates a closure attached to a setInterval, and so it can't be GC'd.

All of that being said, I think I was reacting to a boogeyman - I got an out of memory error after running `npm run unittest:dev` for ~30 minutes, and I thought it was due to setInterval leaking, but I can't seem to reproduce this. I don't know enough about Karma's internals, but everything does seem to get destroyed after the tests run.